templates/membership/join.html.twig

```diff
@@ -11,6 +11,7 @@
 {% block open_graph_image_height 630 %}
 
 {% block stylesheets %}
+    <link type="text/css" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500">
```

Comment:
I guess we don't want to use public CDN for this case no ?

Reply:
removed :D